app/README.md

```diff
@@ -67,7 +69,9 @@ From the `app/` directory:
 
 Alternatively, you can run Storybook in a Docker container:
 
-1. From the root directory run `docker-compose exec nextjs npm run storybook`
+1. From the root directory run `docker compose exec nextjs npm run storybook`
```

Comment:
Seeing an error when I run Storybook in Docker, but seems to happen even when running the previous command, so I think it's unrelated. #108